javaharness/java/arcs/android/AndroidRuntimeSettings.java

```diff
@@ -143,6 +143,7 @@ public boolean useCacheManager() {
    * @param <T> The expected type of returned data.
    * @return the resolved content of property in type T.
    */
+  @SuppressWarnings("RuntimeExec")
   private <T> T getProperty(String property, Function<String, T> converter, T defaultValue) {
```

Comment:
I think this might because this can be used for getting access to prototype properties? If we could read some internal history about the warning I'd feel more comfortable. But for now at least I think its justified.